hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -223,6 +218,22 @@ public void setRowFilter(String row) {
     this.row = row;
   }
 
+  /**
+   * Option to print the row key only in case you just need the row keys from the WAL
+   */
+  public void setOutputOnlyRowKey() {
+    this.outputOnlyRowKey = true;
+  }
+
+  /**
+   * sets the position to start seeking the WAL file
+   * @param position
+   *          initial position to start seeking the given WAL file
+   */
+  public void setPosition(long position) {
```

Comment:
This code seems to be from HBASE-21185. Mind cherry-picking that in a separate commit (in the same PR). Easier to track it that way.

Reply:
@bharathv I didn't add this method or position as part of this commit, it is already there at the end, I just added a java doc and repositioned it just like the master. Do you still want to have it as a separate commit?

Reply:
Ah sorry, I missed that part. No worries then.